UITranslationEstonian/labels-et.txt

```diff
@@ -1082,3 +1082,27 @@ Message_Other_Volcano1=See uus koht on veidi soojem kui eelmine, aga mul on ikka
 Message_Other_Volcano2=<br><br><b>Nüüd ma olen vang! Füüsiliselt koopas ja raudades, vaimselt ühe idee küüsis, ühe õudsa ja möödapääsematu idee küüsis! Mul on ainult üks mõte, kinnisidee. Surma määratud!</b><br><br> 
 GROUP_DESC_GoldenEffigie7=Kuldne kuju
```

Comment:
Change this to 
GROUP_NAME_GoldenEffigie7=Kuldne kuju

Reply:
Done.